standalone/src/models/product/product/product-data/fields-config.ts

```diff
@@ -108,13 +103,6 @@ export const graphqlFieldsConfig: TModelFieldsConfig<TProductDataGraphql> = {
     const metaTitle = metaTitleAllLocales
       ? LocalizedString.resolveGraphqlDefaultLocaleValue(metaTitleAllLocales)
       : undefined;
-    const categoriesRef = model.categories.map<TReferenceGraphql<'category'>>(
-      (category) => ({
-        id: category.id,
-        typeId: 'category',
-        __typename: 'Reference',
-      })
-    );
```

Comment:
We should not remove this.

In case the model has been populated with some categories by the consumers, we still want to create the references automatically.